service/src/main/java/uk/gov/hmcts/reform/fpl/model/ApplicationDocument.java

```diff
@@ -6,22 +6,35 @@
 import lombok.Data;
 import uk.gov.hmcts.reform.fpl.enums.ApplicationDocumentType;
 import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.interfaces.FurtherDocument;
 
 import java.time.LocalDateTime;
+import java.util.List;
 
 @Data
 @Builder(toBuilder = true)
 @AllArgsConstructor
-public class ApplicationDocument {
+public class ApplicationDocument implements FurtherDocument {
     private final DocumentReference document;
     private final ApplicationDocumentType documentType;
     protected LocalDateTime dateTimeUploaded;
     private String uploadedBy;
     private String documentName;
     private String includedInSWET;
+    private List<String> confidential;
 
     @JsonIgnore
     public boolean hasDocument() {
         return document != null;
     }
+
+    @JsonIgnore
+    public boolean isConfidentialDocument() {
+        return confidential != null && confidential.contains("CONFIDENTIAL");
```

Comment:
I'm probably missing something but why does confidential need to be a list? Could it not maybe be boolean?

Reply:
Can see it's already defined like that in the SupportingEvidence bundle so guess that's why